example/src/shared/utils.ts

```diff
@@ -14,3 +18,20 @@ export const getShortUsername = (username: string) => {
 export const checkIfStringContainsOnlyWhitespaces = (val: string) => {
   return !val.trim();
 };
+
+export const checkIfUrl = (val: string) => {
+  return val.startsWith(VIDEOROOM_URL);
+};
+
+export const extractRoomNameFromUrl = (url: string) => {
+  return url.substring(url.indexOf(VIDEOROOM_URL) + VIDEOROOM_URL.length);
+};
+
+export const shouldEnableRoomButton = (username, roomName) => {
```

Comment:
Maybe `checkIfStringContainsOnlyWhitespaces` could check if string is empty too and this function would be simpler and reduced to a single if statement? Also we could rename `checkIfStringContainsOnlyWhitespaces` to `isEmptyStringOrWhitespaces` or something like that to be more clear.
I'm not saying it's wrong putting it here, it's just weird to me putting this function in this file, because `@utils` is supposed to hold functions useful in the whole app, this function is useful only in two specific screens.

Reply:
Removed and renamed.